core/types.go

```diff
@@ -1,114 +1,115 @@
 package core
 
 var (
-	// ActionContainerArchive describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#get-an-archive-of-a-filesystem-resource-in-a-container
+	// ActionContainerArchive describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/get-an-archive-of-a-filesystem-resource-in-a-container
 	ActionContainerArchive = "container_archive"
-	// ActionContainerArchiveExtract describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#extract-an-archive-of-files-or-folders-to-a-directory-in-a-container
+	// ActionContainerArchiveExtract describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/extract-an-archive-of-files-or-folders-to-a-directory-in-a-container
 	ActionContainerArchiveExtract = "container_archive_extract"
-	// ActionContainerArchiveInfo describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#retrieving-information-about-files-and-folders-in-a-container
+	// ActionContainerArchiveInfo describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/retrieving-information-about-files-and-folders-in-a-container
 	ActionContainerArchiveInfo = "container_archive_info"
-	// ActionContainerAttach describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#attach-to-a-container
+	// ActionContainerAttach describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/attach-to-a-container
 	ActionContainerAttach = "container_attach"
-	// ActionContainerAttachWs describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#attach-to-a-container-websocket
+	// ActionContainerAttachWs describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/attach-to-a-container-websocket
 	ActionContainerAttachWs = "container_attach_websocket"
-	// ActionContainerChanges describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#inspect-changes-on-a-container-s-filesystem
+	// ActionContainerChanges describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/inspect-changes-on-a-container-s-filesystem
 	ActionContainerChanges = "container_changes"
-	// ActionContainerCommit describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#create-a-new-image-from-a-container-s-changes
+	// ActionContainerCommit describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.24/#/create-a-new-image-from-a-container-s-changes
 	ActionContainerCommit = "container_commit"
-	// ActionContainerCopyFiles describes https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#copy-files-or-folders-from-a-container
-	ActionContainerCopyFiles = "container_copyfiles"
```

Comment:
@liron-l I see your point. It can stay there for the sake of older daemon versions, even if it throws an error for the new API (that's client problem!).

---
Just for reference Docker has removed this functionality and according to the docs:

> `POST /containers/{name:.*}/copy` is now removed and errors out starting from this API version.

So if you have API v >= 1.24, it *will* return an error regardless of having this line or not! In any other case, it would have been no problem since the API is backward compatible.